lib/Transforms/Scalar/ScalarReplAggregates.cpp

```diff
@@ -75,10 +75,7 @@ namespace {
         ArrayElementThreshold = 8;
       else
         ArrayElementThreshold = AT;
-      if (SLT == -1)
         // Do not limit the scalar integer load size if no threshold is given.
-        ScalarLoadThreshold = -1;
-      else
         ScalarLoadThreshold = SLT;
```

Comment:
It's our practice when editing files that are original to LLVM and not added as part of the DXC project to add comments with ```// HLSL Change``` to make places we've made alterations clear. 

In this case, it might be worth noting that we currently don't use the code in this file. ScalarRepleAggregatesHLSL.cpp is where it's at!